typegate/src/runtimes/wasm.ts

```diff
@@ -2,107 +2,21 @@
 // SPDX-License-Identifier: Elastic-2.0
 
 import { Runtime } from "./Runtime.ts";
-import * as native from "native";
-import { Resolver, RuntimeInitParams } from "../types.ts";
-import { nativeResult } from "../utils.ts";
-import { ComputeStage } from "../engine/query_engine.ts";
+import { RuntimeInitParams } from "../types.ts";
 import { registerRuntime } from "./mod.ts";
-import * as ast from "graphql/ast";
-import { Materializer } from "../typegraph/types.ts";
-import { Typegate } from "../typegate/mod.ts";
+import { WasmRuntimeData } from "../typegraph/types.ts";
+import { WasmRuntimeReflected } from "./wasm/reflected.ts";
+import { WasmRuntimeWire } from "./wasm/wire.ts";
 
 @registerRuntime("wasm")
```

Comment:
Not here, I've mapped both wire and reflected wasm runtimes to `wasm` as their runtime name. Disambiguation is done on the runtime data instead. I'm unsure if this is the a right decision and am willing to refactor to distinct names.